server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -175,12 +175,14 @@ export class ScribeLambdaFactory
 				}
 			}
 
+			const isEphemeralContainer = document.isEphemeralContainer;
```

Comment:
>document.isEphemeralContainer [](http://example.com/codeflow?start=32&length=29)

preserve this?
document?.isEphemeralContainer

Reply:
Updated to keep the optional chaining